webpack/webpack.prod.js

```diff
@@ -15,8 +15,17 @@ const getOptions = (env) => {
   if (!isEs5(env)) return {};
 
   return {
+    sourceType: "unambiguous",
     babelrc: false,
-    presets: ["@babel/env"],
+    presets: [
+      [
+        "@babel/env",
+        {
+          corejs: "3",
```

Comment:
I've noticed even though we have babel transpiling modern code, we still need polyfills for functions that do not supported in IE 11 (Array.includes, string.startsWith). I've used babel-polyfill in the past for polyfilling, but it's deprecated now in favor of core-js. With this config polyfills will be added automatically when a feature is unsupported in target environment (in this case target env is legacy browsers)

Explanation is here: https://stackoverflow.com/a/56505264

Reply:
Thanks for the comment! Could you include a shorter version of this comment inside the JS file directly?